packages/remix-testing/create-remix-stub.tsx

```diff
@@ -47,7 +42,7 @@ type RemixStubOptions = {
   remixConfigFuture?: Partial<FutureConfig>;
 };
 
-export function createRemixStub(routes: AgnosticDataRouteObject[]) {
+export function createRemixStub(routes: (RouteObject | DataRouteObject)[]) {
```

Comment:
Unsure if `DataRouteObject` is necessary here since it's just a narrower version of `RouteObject`